src/lib.rs

```diff
@@ -1571,14 +1537,14 @@ impl Connection {
             [0; std::mem::size_of::<QuicStatistics>()];
         let stat_size_mut = std::mem::size_of::<QuicStatistics>();
         unsafe {
-            ((*APITABLE).get_param)(
+            Api::get_param(
                 self.handle,
                 PARAM_CONN_STATISTICS,
                 (&stat_size_mut) as *const usize as *const u32 as *mut u32,
-                stat_buffer.as_mut_ptr() as *const c_void,
+                stat_buffer.as_mut_ptr() as *mut c_void,
             )
-        };
-
+        }
+        .expect("fail to get stats");
```

Comment:
For `get_perf`, you chose to return an error, while you assert here.
Is there a reason for the difference? Or should we be consistent?

Reply:
The original API does not return error so I kept the same. It maybe infallible? Or I can change it.

Reply:
Made all of them return Result.